docs/getting_started/tfm_integration_guide.rst

```diff
@@ -29,6 +29,8 @@ The hardware platforms currently supported are:
 - CoreLink SSE-200 Subsystem for MPS3 (AN524)
 - DesignStart FPGA on Cloud: Cortex-M33 based platform (SSE-200_AWS)
 - STM32L5xx: Cortex-M33 based platform (STM32L562 and STM32L552 socs)
+- nRF9160 development kit (Cortex-M33)
```

Comment:
Could you align with the names used in upstream Zephyr ?

Reply:
Like so?
```suggestion
- nrf9160dk_nrf9160 (Cortex-M33)
```

Reply:
I think it should be "nRF9160 DK". That's what is used in Nordic infocenter and [website](https://www.nordicsemi.com/Software-and-Tools/Development-Kits/nRF9160-DK), and also in Zephyr's [documentation](https://docs.zephyrproject.org/latest/boards/arm/nrf9160dk_nrf9160/doc/index.html).

Reply:
agree